app/views/docs/getting-started-for-server.phtml

```diff
@@ -382,7 +382,9 @@ let client = Client()
     <li>
         <h3>Node.js</h3>
         <div class="ide margin-top-small" data-lang="nodejs" data-lang-label="Node.js SDK">
-            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const users = new Users(client);
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const { Client, ID, Users } = require('node-appwrite');
```

Comment:
This is the last thing I'll ask for I swear. Let's remove the imports that are not used
```suggestion
            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const { ID, Users } = require('node-appwrite');
```